spec/models/admin_dashboard_data_spec.rb

```diff
@@ -2,6 +2,32 @@
 
 describe AdminDashboardData do
 
+  describe "adding new checks" do
+    it 'calls the passed block' do
+      called = false
+      AdminDashboardData.add_problem_check do
+        called = true
+      end
+
+      AdminDashboardData.fetch_problems
+      expect(called).to eq(true)
+    end
+
+    it 'calls the passed method' do
+      $test_AdminDashboardData_global = false
```

Comment:
You are not tearing down the stuff you add to `AdminDashboardData` in these methods. Other tests will have them depending on when this test was added.